src/AisParser/Payload.cs

```diff
@@ -139,11 +189,19 @@ public string ReadString(int startIndex, int length)
 
             return value.Trim();
         }
+        /*public void WriteString(string var, int length)
```

Comment:
Remove commented out code

Reply:
I left this as a placeholder as I wasn't exactly clear on taking a string value and writing it to AIS stream of bits. If you have a quick look at ReadString, it should perform the reverse of that function.

Reply:
Side note, this will be necessary for converting class's with "String" fields to binary.